pkg/kubelet/kubelet.go

```diff
@@ -444,7 +444,7 @@ func NewMainKubelet(kubeCfg *kubeletconfiginternal.KubeletConfiguration,
 		nodeLister = kubeInformers.Core().V1().Nodes().Lister()
 		nodeHasSynced = func() bool {
 			if kubeInformers.Core().V1().Nodes().Informer().HasSynced() {
-				klog.Infof("kubelet nodes sync")
+				klog.V(1).Infof("kubelet nodes sync")
```

Comment:
I'm not sure if V(1) is the right log level for this but WFM

Reply:
Lets just remove this line completely.  It is the expected case and doesn't offer a lot of debugging value.  The line with debugging value is 450 indicating the informer has not yet synced.  Thans!

Reply:
Thanks for your reviews @sjenning @ehashman , fixed.